src/main/kotlin/fr/yvernault/rexa/model/Assessor.kt

```diff
@@ -0,0 +1,21 @@
+package fr.yvernault.rexa.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+import fr.yvernault.rexa.XnatGlossary
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+data class Assessor(
+        @JsonProperty("ID") val id: String,
```

Comment:
You should try to get as well the information on the subjectId and label and session label.